src/models/genre.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (connection, DataTypes) => {
+    const schema = { 
+        genre: {
+            type: DataTypes.STRING,
+            unique: { args: true, msg: "Genre already on the system"},
+            allowNull: false,
+            validate: {
+                notEmpty: { args: true, msg: "Please enter a genre" },
+                notNull: { args: true, msg: "Please enter a genre" },
+        },
+    }
+} 
+const GenreModel = connection.define("Genre", schema);
+return GenreModel
+}
```

Comment:
You are using 4 spaces for indentation instead of 2 which is what you have used throughout the project
```
  const schema = {
    genre: {
      type: DataTypes.STRING,
      unique: { args: true, msg: "Genre already on the system" },
      allowNull: false,
      validate: {
        notEmpty: { args: true, msg: "Please enter a genre" },
        notNull: { args: true, msg: "Please enter a genre" },
      },
    },
  };
  const GenreModel = connection.define("Genre", schema);
  return GenreModel;
};
```